File: server/routes/api/reports.js
const localReports = require('./local/reports')
const jiraReports = require('./jira/reports')

module.exports = function (router) {
  router.get('/reports/perf-stats', function (req, res) {
    if (req.settings && req.settings.useJira) {
      return jiraReports.perfStats(req, res)
    }
    return localReports.perfStats(req, res)
  })

  router.get('/reports/epicremaining', function (req, res) {
    if (req.settings && req.settings.useJira) {
      return jiraReports.epicRemaining(req, res)
    }
    res.send('Nothing to see here')
  })
}

File: server/api/reports.py
from .app import API_APP


@API_APP.route('/reports/perf-stats', methods=['GET'])
def get_perf_stats_report():
    return ('', 200)


@API_APP.route('/reports/epicremaining', methods=['GET'])
def get_epic_remaining_report():
    return ('', 200)
